assets/scss/components/_footer.scss

```diff
@@ -0,0 +1,31 @@
+.footer {
+	&__list-bigwrap {
+		display: flex;
+    	justify-content: center;
```

Comment:
watch the indentation.  I won't make this comment any more, but it's definitely something to work on.